localstack/services/internal.py

```diff
@@ -288,7 +308,7 @@ def add_default_routes(self):
             self.add("/diagnose", DiagnoseResource())
 
     def add(self, path, *args, **kwargs):
-        super().add(f"{constants.INTERNAL_RESOURCE_PATH}{path}", *args, **kwargs)
+        return super().add(f"{constants.INTERNAL_RESOURCE_PATH}{path}", *args, **kwargs)
```

Comment:
@thrau I added this return so that we could remove the rule in the test, I suppose it keeps the same signature as the router so this should be good?

Reply:
yep sounds good!

side note: i think we should get rid of this Resource Router thing all together and just add `/_localstack` rules to the regular edge `ROUTER`. totally out of scope though. just making a note.